src/docs/Foundations/Colors/index.mdx

```diff
@@ -124,11 +124,11 @@ Secondary Purple is used for illustration purposes.
 
 In this system, every color has a `dark` and a `light` variant that can be used to adapt to different usages from text to illustrations.
 
-In order to have this flexibility we designed color swatches based on **9 values** for primary and secondary colors. It means that you can use 9 values of the same color to fit your design needs.
+To have this flexibility, we designed color swatches based on **9 values** for primary and secondary colors. It means that you can use 9 values of the same color to fit your design needs.
 
-> HSLA is a way to define colors based on three properties : **Hue**, **Saturation**, **Light** and **Alpha**. We decided to use this standard to have more consistency between colors. It is also easier to relate colors to each other when they are based on HSLA.
+> HSLA is a way to define colors based on three properties: **Hue**, **Saturation**, **Light**, and **Alpha**. We decided to use this standard to have more consistency between colors. It is also easier to relate colors to each other when they are based on HSLA.
```

Comment:
I think we can delete this part
@caqueste, we're not making rockets fly right?

Reply:
@nherwegh @mohamedMok I think this part can save us from the covid..
Let's delete it haha